api/v1/model/src/commonMain/kotlin/EnvironmentVariableDeclaration.kt

```diff
@@ -34,5 +34,8 @@ data class EnvironmentVariableDeclaration(
     val name: String,
 
     /** The name of the secret that contains the value of the variable. */
-    val secretName: String
+    val secretName: String? = null,
+
+    /** The value of the environment variable. */
```

Comment:
The docs should also describe that it is an error to set both `secretName` and `value`. Same in the file below.